media-sound/teamspeak-server-bin/teamspeak-server-bin-3.1.1.ebuild

```diff
@@ -0,0 +1,134 @@
+HOMEPAGE="https://www.teamspeak.com/"
+SRC_URI="amd64? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_amd64-${PV}.tar.bz2 )
+	x86? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_x86-${PV}.tar.bz2 )"
+
+LICENSE="teamspeak3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc mysql tsdns"
+
+RDEPEND="mysql? ( dev-db/mariadb-connector-c )"
+
+RESTRICT="fetch mirror strip"
+
+S="${WORKDIR}/teamspeak3-server_linux_${ARCH}"
+
+pkg_nofetch() {
+	elog "Please download ${A} from ${HOMEPAGE}"
+	elog "and place it in your DISTDIR directory."
+}
+
+pkg_setup() {
+	enewuser teamspeak
+}
+
+src_install() {
+	local TS3_DIR="teamspeak3-server"
+	local TS3_UG="teamspeak"
+
+	# Accept license
+	touch "${T%/}"/.ts3server_license_accepted || die
+	insinto "/opt/${TS3_DIR}"
+	doins "${T}"/.ts3server_license_accepted
+
+	# Install binary and wrapper
+	exeinto "/opt/${TS3_DIR}"
+	doexe "ts3server"
+	insinto "/opt/${TS3_DIR}"
+	doins "${FILESDIR}/ts3server-bin"
+	dodir "/opt/bin"
+	dosym "../$TS3_DIR/ts3server-bin" "/opt/bin/ts3server"
+
+	# Install libs
+	insinto "/opt/${TS3_DIR}"
+	doins "libts3db_sqlite3.so"
+
+	# Install sql
+	insinto "/opt/${TS3_DIR}/sql"
+	doins "sql"/*.sql
+	insinto "/opt/${TS3_DIR}/sql/create_sqlite"
+	doins "sql/create_sqlite"/*.sql
+
+	# Install config
+	insinto "/etc/${TS3_DIR}"
+	doins "${FILESDIR}/ts3server.ini"
+
+	# Install init script and systemd unit
+	newinitd "${FILESDIR}/teamspeak.initd" teamspeak3-server
+	systemd_newunit "${FILESDIR}/teamspeak.service" teamspeak3-server.service
+	systemd_newtmpfilesd "${FILESDIR}/teamspeak.tmpfiles" teamspeak3-server.conf
+
+	# Install optional mysql
+	if use mysql; then
+		insinto "/etc/${TS3_DIR}"
+		doins "${FILESDIR}/ts3db_mariadb.ini"
+
+		insinto "/opt/${TS3_DIR}"
+		doins "libts3db_mariadb.so"
+
+		insinto "/opt/${TS3_DIR}/sql/create_mariadb"
+		doins "sql/create_mariadb/"*.sql
+
+		insinto "/opt/${TS3_DIR}/sql/updates_and_fixes"
+		doins "sql/updates_and_fixes/"*.sql
+	fi
+
+	# Install optional docs
+	if use doc; then
+		local HTML_DOCS="doc/serverquery/*"
+
+		docinto "serverquery"
+		dodoc "serverquerydocs"/*.txt
+
+		docinto "ts3server"
+		dodoc "doc"/*.txt
+	fi
+
+	# Install tsdns
+	if use tsdns; then
+		exeinto "/opt/${TS3_DIR}"
+		doexe "tsdns/tsdnsserver"
+		dodir "/opt/bin"
+		dosym "../$TS3_DIR" "/opt/bin/tsdnsserver"
+
+		insinto "/etc/${TS3_DIR}"
+		doins "tsdns/tsdns_settings.ini.sample"
+
+		docinto "tsdns"
+		dodoc "tsdns/README" "tsdns/USAGE"
+	fi
+
+	# Install docs
+	einstalldocs
+
+	# Keep directories
+	keepdir "/etc/${TS3_DIR}"
+	keepdir "/var/log/${TS3_DIR}"
+
+	# Protect config
+	if use mysql; then
+		echo "CONFIG_PROTECT=\"/etc/${TS3_DIR}/ts3server.ini /etc/${TS3_DIR}/ts3server_mariadb.ini\"" > "${T}"/99${TS3_DIR}
+	else
+		echo "CONFIG_PROTECT=\"/etc/${TS3_DIR}/ts3server.ini\"" > "${T}"/99${TS3_DIR}
+	fi
+        doenvd "${T}"/99${TS3_DIR}
+
+	# Set permissions
+	fowners -R ${TS3_UG}:${TS3_UG} "/etc/${TS3_DIR}"
```

Comment:
You can pass multiple dirs to `fowners`,

Reply:
Okay, will change it. I personally like to have each directory in one line.